fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -123,6 +123,7 @@ DEFERRED: 'DEFERRED';
 NTILE: 'NTILE';
 DESC: 'DESC';
 DESCRIBE: 'DESCRIBE';
+DISABLE: 'DISABLE';
```

Comment:
mark them reserved words ?

Reply:
> mark them reserved words ?

this will affect users' behavior if they use disable as column name…I prefer not to do this.